noir-projects/noir-contracts/contracts/router_contract/src/utils.nr

```diff
@@ -0,0 +1,15 @@
+use aztec::protocol_types::constants::ROUTER_ADDRESS;
+use aztec::context::private_context::PrivateContext;
+use crate::Router;
+
+// docs:start:helper_router_functions
+pub fn privately_check_timestamp(operation: u8, value: u64, context: &mut PrivateContext) {
+    Router::at(ROUTER_ADDRESS).check_timestamp(operation, value).call(context);
+}
+
+pub fn privately_check_block_number(operation: u8, value: Field, context: &mut PrivateContext) {
+    // docs:start:enqueueing
+    Router::at(ROUTER_ADDRESS).check_block_number(operation, value).call(context);
+    // docs:end:enqueueing
+}
```

Comment:
I'd add some inline docs here explaining that these functions preserve privacy by performing this check via the router, since it'd otherwise reveal the calling contract. If not a reader would need to not only go to the docsite to find more information, but they wouldn't even know where to search (because they don't know where the `helper_router_functions` tag was included).

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/8384/commits/4ed01efe23ec41d6e4f1549ae1459db6c6f425b2